src/ParallelAlgorithms/ApparentHorizonFinder/Events/FindApparentHorizon.hpp

```diff
@@ -78,7 +78,7 @@ class FindApparentHorizon : public Event {
                   const tnsr::iaa<DataVector, 3>& phi,
```

Comment:
Not really an accurate commit message, since this also removes some template parameters (and as a result changes the values passed in some tests).  No objection to the actual content of the change.